### ISSUE TITLE
track Units

### ISSUE THREAD

**Issue by natillasol** [cambiatus/frontend#52]
**Is your feature request related to a problem? Please describe.**
When creating an offer, you have the option of tracking units. This is not very clear to me at first hand, but since I know the project I can make an idea of it. However, we need something clearer.

**Describe the solution you'd like**

Maybe say: "Would you like to track how many units have been sold?"

OR

Have a clarification box that explains the function of this selection. 

**Additional context**
<img width="1440" alt="Screen Shot 2019-11-13 at 13 10 36" src="https://user-images.githubusercontent.com/57719975/68795892-767adf80-0617-11ea-94f3-62f97d5cbb4b.png">

**Comment by lucca65** [comment]
@natillasol this is also something that should be addressed by the UX-UI team. I'll close it and forward it to them!

**Comment by heltonlr** [comment]
@natillasol 
We are considering your feedback on the new interfaces, thank you very much.
<img width="366" alt="Captura de Tela 2019-11-13 às 21 12 13" src="https://user-images.githubusercontent.com/39558132/68815708-eeb6c480-065a-11ea-8b27-1b8c2354bea8.png">

**Comment by zacck** [comment]
Hmm @lucca65  I feel as though we should icebox these issues rather than close them, that way once the xu/ui team is done with there part we can jump on these available issues

**Comment by lucca65** [comment]
Hey @zacck why we should track this as an issue if we don't even know yet how it is going to look like after the UX team handles it? 

In any case we will have tasks to implement new design changes as they come. Thats the reason to not keep this issue opened since the solution might actually be very different (for instance a complete redesign). This will require us to do a lot of back tracking or simply having tons of issues in the icebox

This is my reasoning behind this decision, any thoughts?

**Comment by natillasol** [comment]
@zacck @lucca65 @heltonlr  Should we have a separate "board" (not sure of the name) in Github for the features?

**Comment by lucca65** [comment]
@natillasol we do

https://github.com/orgs/cambiatus/projects/2

But the design team has its own task management

**Comment by zacck** [comment]
@lucca65  hmmm seems reasonable ...

**Comment by heltonlr** [comment]
I have a backlog column:

<img width="322" alt="Captura de Tela 2019-11-14 às 12 50 28" src="https://user-images.githubusercontent.com/39558132/68872811-662d3800-06dd-11ea-982c-08afd4c3bf62.png">

**Comment by natillasol** [comment]
Excuse my ignorance, but is it possible to have everything in one app? in Github?

**Comment by heltonlr** [comment]
> Excuse my ignorance, but is it possible to have everything in one app? in Github?

Yes I think. But we would have to differentiate by tag (improvement / bug), I don't mind being all in Github.

**Comment by natillasol** [comment]
I honestly find it easier, so we (users) don't have to jump between two different platforms

**Comment by heltonlr** [comment]
> I honestly find it easier, so we (users) don't have to jump between two different platforms

Cool, so we can create an **"Improvement"** tag in Git, if you agree @zacck and @lucca65

**Comment by lucca65** [comment]
@natillasol we use GitHub to track development tasks. Right now @heltonlr is using Asana for the UX team. 

He would need to start using GitHub to track UX issues. Which is entirely possible, of course. That being said, I don't think it is productive to create tags in this repo that aren't related to the development tasks!

Think of it as different boards for different contexts. Lets try to use the best tool for the context we need

**Comment by natillasol** [comment]
so, what should I don then?

**Comment by lucca65** [comment]
This is for sure something that should be handled by the design! So it is up to @heltonlr how he is going to track his tasks!
